cmds/ocm/common/output/table.go

```diff
@@ -9,8 +9,8 @@ import (
 
 func FormatTable(ctx Context, gap string, data [][]string) {
 	columns := []int{}
-	max := 0
-	maxtitle := 0
+	maxValue := 0
+	maxTitle := 0
 
```

Comment:
Is all this a linter problem, or why have you touched so many files not related to the PR. By he way, the linter version specified in the requirements does NOT complain about this. Why did you use another one.

But anyway, in this case here, the change is really useful, because there are different max values. But all the other locations.....

Reply:
Because `max` and `min` are now reserved functions. :)

Reply:
Yah, that was an interesting choice... could have been in math or something.